src/AnonymizationStep/AnonymizationStep.tsx

```diff
@@ -63,24 +63,20 @@ function summaryDescriptions(summary: AnonymizationSummary) {
 
 function AnonymizationSummary({ result: { summary }, loading }: CommonProps) {
   return (
-    <>
+    <div className="AnonymizationSummary notebook-step">
       <NotebookNavAnchor step={NotebookNavStep.AnonymizationSummary} status={loading ? 'loading' : 'done'} />
       <Title level={3}>Anonymization summary</Title>
       {summary === emptySummary ? (
-        <div className="AnonymizationSummary loading notebook-step">
+        <div className="AnonymizationSummary loading">
```

Comment:
To match with our naming style, this should now be something like `AnonymizationSummary-spinner`. Maybe div can be removed and collapsed with the `<Space>` below.

Reply:
I have been basing off of `<div className="SchemaLoadStep notebook-step loading">`, should that be updated as well?